src/render/pic.rs

```diff
@@ -0,0 +1,78 @@
+//! PIC rendering support.
+//!
+//! # Example
+//!
+//! ```
+//! use qrcode::QrCode;
+//! use qrcode::render::pic;
+//!
+//! let code = QrCode::new(b"Hello").unwrap();
+//! let pic = code.render::<pic::Color>().build();
+//! println!("{}", pic);
+
+#![cfg(feature = "pic")]
+
+use std::fmt::Write;
+use std::marker::PhantomData;
+
+use crate::render::{Canvas as RenderCanvas, Pixel};
+use crate::types::Color as ModuleColor;
+
+/// A PIC color.
+#[derive(Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Color<'a>(pub &'a str);
+
+impl<'a> Pixel for Color<'a> {
+    type Canvas = Canvas<'a>;
+    type Image = String;
+
+    fn default_color(color: ModuleColor) -> Self {
+        Color(color.select("#000000", "#ffffff"))
+    }
+}
+
+#[doc(hidden)]
+pub struct Canvas<'a> {
+    pic: String,
+    marker: PhantomData<Color<'a>>,
+}
+
+impl<'a> RenderCanvas for Canvas<'a> {
+    type Pixel = Color<'a>;
+    type Image = String;
+
+    fn new(width: u32, height: u32, dark_pixel: Color<'a>, light_pixel: Color<'a>) -> Self {
+        Canvas {
+            pic: format!(
+                concat!(
+                    r#"command ".if !mqrcode_fg .defcolor qrcode_fg rgb {fg}""#,
+                    "\n",
+                    r#"command ".if !mqrcode_bg .defcolor qrcode_bg rgb {bg}""#,
+                    "\n",
```

Comment:
I don't see any point in supporting overriding the color externally. Remove it.

```suggestion
                    "qrcode_fg = {fg}\n",
                    "qrcode_bg = {bg}\n",
```

Reply:
ℹ️ GNU PIC does not support defining colors on its own, but rather relies on the color definition capabilities of GNU roff.

To remove support for overriding colors only the `.if !mqrcode_[fb]g` needs to be removed.

I'll address this shortly.

Reply:
The point of being able to override the color is to be able to use the same QR-Code source PIC file and change the color to match the theme of the document it is included in without the need to re-generate the QR-Code or store other generated  version of the same QR-Code in separate files.

What are reasons against supporting it?

Reply:
The content of the `command` command is post-processor-dependent. Using `.if` makes the generated file impossible to be used in TeX mode or standalone implementations like pikchr. 

I don't find "change the color to match the theme of the document" to be a good argument to special-case PIC here. The same argument applies to SVG and PNG after all. If you really need these special treatment you could always prepend these magic commands before writing the file, or edit the file after it is generated.

Reply:
Excellent objection, @kennytm, thank you very much for that 😃
I had not considered the portability of the generated PIC code!

At first glance it appears to me that [pikchr](https://pikchr.org) takes much inspiration from PIC, but makes not guarantees on compatibility.

Since "Colored Objects [are] a GNU extension" (see section "6.10 Colored Objects" in ["Making Pictures with GNU PIC"](https://archive.org/details/manualzilla-id-5804556)) and pikchr has a slightly different syntax for colored objects I suggest to remove color support altogether.

What do you think, @kennytm?

Reply:
> Since "Colored Objects [are] a GNU extension" (see section "6.10 Colored Objects" in ["Making Pictures with GNU PIC"](https://archive.org/details/manualzilla-id-5804556)) and pikchr has a slightly different syntax for colored objects I suggest to remove color support altogether.

that's fine for me :)